src/client/components/equalizer/canvasEqualizer/canvasEqualizer.css

```diff
@@ -0,0 +1,3 @@
+/* .graphicCanvas{
```

Comment:
never use comments for code, only for describe functionality

Reply:
done